packages/vuetify/src/directives/index.ts

```diff
@@ -1,3 +1,6 @@
+// @ts-nocheck
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
Remove

Reply:
See this file for other directives that are already done and shouldn't be skipped